TestProjects/PerceptionHDRP/Packages/packages-lock.json

```diff
@@ -81,7 +81,7 @@
         "com.unity.collections": "0.9.0-preview.6",
         "com.unity.nuget.newtonsoft-json": "1.1.2",
         "com.unity.render-pipelines.core": "7.1.6",
-        "com.unity.simulation.capture": "0.0.10-preview.19",
+        "com.unity.simulation.capture": "0.0.10-preview.20",
```

Comment:
I don't have a problem with this, but just wanted to make sure that we wanted to upgrade capture to 20 in this fix, and not just happening by accident.

Reply:
Yea, good eye. this packages-lock.json is an auto-generated file that should have already been updated. It doesn't actually represent a change in the dependencies.